index.html

```diff
@@ -70,4 +70,166 @@ <h3 class="supporting-text">I’am a Software Developer</h3>
+                            sign-ups required. has been the industry's standard</p>
+                            <ul class="lang">
+                                <li><a href="@" class="btn btn-car">html</a></li>
+                                <li><a href="@" class="btn btn-car">bootstrap</a></li>
+                                <li><a href="@" class="btn btn-car">Ruby</a></li>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+
+                <div class="card item5">
+                    <div class="text-block">
+                        <h3 class="new">Profesional Art<br>Printing Data</h3>
+                        <p class="card-description">A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required. has been the industry's standard</p>
+                            <ul class="lang">
+                                <li><a href="@" class="btn btn-car">html</a></li>
+                                <li><a href="@" class="btn btn-car">bootstrap</a></li>
+                                <li><a href="@" class="btn btn-car">Ruby</a></li>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+
+                <div class="card item6">
+                    <div class="text-block">
+                        <h3 class="new">Profesional Art<br>Printing Data</h3>
+                        <p class="card-description">A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required. has been the industry's standard</p>
+                            <ul class="lang">
+                                <li><a href="@" class="btn btn-car">html</a></li>
+                                <li><a href="@" class="btn btn-car">bootstrap</a></li>
+                                <li><a href="@" class="btn btn-car">Ruby</a></li>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+            </div>
+            </div>
+    </section>
+
+        <section class="about-section">
+            <div class="about">
+                <h2 class="spaced">About me</h2>
+                <p class="about-descript">Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                    <button class="abt-btn">Get My Resume</button>
+                    <img class="divider" src="images/div.png" alt=""/>
+            </div>
```

Comment:
- [ ] I have noticed the image with the class of divide is slightly aligned to the right yet originally in the figma design the divider is aligned in the center. The tilt is caused by this style 👉🏻 `.divider { margin-left: 40px; } `. So how about setting the margin-left to `0` or completely deleting the style 😏? Doing this will correct its alignment.

- [ ] Below is the comparison of your design and how it should look like. I highly recommend that you add a background image as seen in the figma design so that you maintain consistency in the top and work, about sections

| Your design | Figma design |
| --- | --- |
| <img width="282" alt="1232" src="https://user-images.githubusercontent.com/4247740/144014566-d0af3461-1ba0-404e-96dc-c2ddf42d1630.PNG"> | <img width="283" alt="123" src="https://user-images.githubusercontent.com/4247740/144014537-e7acf79e-1829-42a6-b53f-ed4d51137edf.PNG"> |

Reply:
Honestly, I would be grateful if you can help with how I can add the image to the background because I have tried.